notification/src/main/java/com/kyriba/conference/notification/services/SmsNotificationRequestService.java

```diff
@@ -0,0 +1,32 @@
+package com.kyriba.conference.notification.services;
+
+import com.kyriba.conference.notification.domain.dto.NotificationStatus;
+import com.kyriba.conference.notification.domain.dto.SmsNotificationMessage;
+import com.kyriba.conference.notification.domain.NotificationInfo;
+import com.kyriba.conference.notification.domain.SmsNotification;
+import com.kyriba.conference.notification.dao.SmsNotificationRepository;
+import lombok.RequiredArgsConstructor;
+import org.springframework.amqp.rabbit.AsyncRabbitTemplate;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+
+@Service
+@RequiredArgsConstructor(onConstructor = @__(@Autowired))
+public class SmsNotificationRequestService
+{
+  private final SmsNotificationRepository smsNotificationRepository;
+  private final AsyncRabbitTemplate asyncRabbitTemplate;
+
+  @Transactional
+  public NotificationStatus sendNotification(SmsNotificationMessage smsNotificationMessage) {
```

Comment:
All 3 sending services are completely the same. It seems you need to create appropriate abstraction in domain model.